frontend/karma.conf.js

```diff
@@ -26,12 +26,6 @@ module.exports = function (config) {
     logLevel: config.LOG_INFO,
     autoWatch: true,
     browsers: ['Chrome'],
-    singleRun: false,
-    customLaunchers: {
```

Comment:
Could you please rebase onto the latest master? I think we want to keep these changes.

Reply:
This is no issue with the history, I accidentally overwrote it - I will reintroduce the configuration to `karma.conf.js` manually.